modules/react/select/stories/Select.stories.mdx

```diff
@@ -0,0 +1,113 @@
+import {DisabledOption} from './examples/DisabledOption';
+import {Error} from './examples/Error';
+import {ErrorStandalone} from './examples/ErrorStandalone';
+import {Grow} from './examples/Grow';
+import {GrowStandalone} from './examples/GrowStandalone';
+import {LabelPosition} from './examples/LabelPosition';
+import {RefForwarding} from './examples/RefForwarding';
+import {Required} from './examples/Required';
+import {Standalone} from './examples/Standalone';
+
+<Meta title="Components/Inputs/Select/React" component={Select} />
+
+# Canvas Kit Select
+
+Select Drop-downs allow users to choose one option from a list of items in a Drop-down Menu.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+## Default Example
+
+Select should be used in tandem with Form Field to meet accessibility standards.
+
+<ExampleCodeBlock code={Default} />
+
+If necessary, Select can also be used on its own.
+
+<ExampleCodeBlock code={Standalone} />
+
+## Disabled
+
+Set the `disabled` prop of the Select to prevent users from interacting with it.
+
+<ExampleCodeBlock code={Disabled} />
+
+You may also set the `disabled` prop of the Select Option to prevent users from selecting that
+Option. Note that the value of a Select may be programmatically set to a disabled Option by setting
+the Select's `value` prop to the value of the disabled Option. The disabled Option, however, will
+remain unselectable by the user.
+
+<ExampleCodeBlock code={DisabledOption} />
+
+## Label Position
+
+Set the `labelPosition` prop of the wrapping Form Field to designate the position of the label
+relative to the Select. `labelPosition` accepts the following values:
+
+- `FormField.LabelPosition.Top` (Default)
+- `FormField.LabelPosition.Left`
+
+<ExampleCodeBlock code={LabelPosition} />
+
+## Required
+
+Set the `required` prop of the wrapping Form Field to `true` to indicate that it's required. Labels
+for required fields are suffixed by a red asterisk.
+
+<ExampleCodeBlock code={Required} />
+
+## Grow
+
+Set the `grow` prop of the wrapping Form Field to `true` to configure the Select to expand to the
+width of its container.
+
+<ExampleCodeBlock code={Grow} />
+
+The `grow` prop may also be applied directly to the Select if Form Field is not being used.
+
+<ExampleCodeBlock code={GrowStandalone} />
+
+## Ref Forwarding
+
+Select supports [ref forwarding](https://reactjs.org/docs/forwarding-refs.html). It will forward
+`ref` to its underlying `<select>` element.
+
+<ExampleCodeBlock code={RefForwarding} />
+
+## Error States
+
+### Alert
+
+Set the `error` prop of the wrapping Form Field to `FormField.ErrorType.Alert` and provide an
+explanation using the `hintText` prop to set the Select to the Alert state.
+
+<ExampleCodeBlock code={Alert} />
+
+The Alert state may be applied directly to the Select using the static `Select.ErrorType.Alert`
+value if Form Field is not being used.
+
+<ExampleCodeBlock code={AlertStandalone} />
```

Comment:
I'm not sure we need all of these standalone stories for props that are available on both `FormField` and the base component. They make the sidebar quite busy. I think the sentence above is enough and it doesn't really require an extra example.

```suggestion
The Alert state may be applied directly to the `Select` component directly using the static `Select.ErrorType.Alert`
value if `FormField` is not being used.

<ExampleCodeBlock code={AlertStandalone} />
```

Reply:
Agreed, I've removed all Standalone stories (`Standalone`, `Alert Standalone`, `Error Standalone`, etc.) throughout all the Input component MDXes.